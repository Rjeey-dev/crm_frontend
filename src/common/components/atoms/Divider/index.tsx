import React from 'react';

function Divider() {
    return <div className='dropdown-divider'/>;
}

export default Divider;
