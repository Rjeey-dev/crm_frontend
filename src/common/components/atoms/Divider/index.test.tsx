import { shallow } from 'enzyme';
import React from 'react';

import Divider from './index';

it("Renders <Divider/> with correct properties", () => {
    const output = shallow(
        <Divider/>
    ).html();

    expect(output).toMatchSnapshot();
});
