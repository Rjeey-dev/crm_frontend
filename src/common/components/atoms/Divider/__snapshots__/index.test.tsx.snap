// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders <Divider/> with correct properties 1`] = `"<div class=\\"dropdown-divider\\"></div>"`;
